packages/ses/src/permits.js

```diff
@@ -281,6 +282,7 @@ export const AsyncFunctionInstance = {
 
 // Aliases
 const fn = FunctionInstance;
+const hermesFn = { ...FunctionInstance, prototype: 'undefined' }; // Bypass Hermes bug, fixed in: https://github.com/facebook/hermes/commit/00f18c89c720e1c34592bb85a1a8d311e6e99599
```

Comment:
Likewise. Is there an issue for this Hermes bug?

Reply:
yep the issue for concise methods is
- https://github.com/facebook/hermes/issues/1371

for concise (arrow) functions [no issue](https://github.com/facebook/hermes/issues?q=prototype+arrow+function) was raised only commit https://github.com/facebook/hermes/commit/c42491de94aff479e5e83c073eff96a6261da080